.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
@pulpopurpura I'm not sure why we have this `.idea` directory and the corresponding files but we should remove them from the commit and I'd recommend adding them to your global gitignore so they don't get added back in automatically: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore